### PR TITLE
Feature/admin dashboreds

### PR DESCRIPTION
**Pull request by Tommmy1018**
・First commit dashboard
・Deleted log.txt

### CODE REVIEW

File: resources/views/admin/dashboards.blade.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
+@extends('layouts.app')
+
```
Comment (kredo-school) [inline resources/views/admin/dashboards.blade.php]: create a section here like

```
@section('styles')
    // ... add the code from line number 7 and line number 8 here and remove them inside `content` section
@endsection
```

and...
in your app.blade.php

and `@yield('styles')` before the `</head>` end tag

File: resources/views/admin/dashboards.blade.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            <h4>Reservations</h4>
+
+        </div>
+    </div>
+
+    {{-- Nationality --}}
+    <div class="container border">
+        <h4>Nationality</h4>
+
+        {{-- graph area --}}
```
Comment (kredo-school) [inline resources/views/admin/dashboards.blade.php]: since we will be using the graph here please add a dummy graph here...

**Review by kredo-school — CHANGES_REQUESTED** [review]

File: resources/views/admin/dashboards.blade.php (hunk 2 above)
Comment (Tommmy1018) [reply resources/views/admin/dashboards.blade.php]: does putting the graph mean pic? or using canvas tag?

File: resources/views/admin/dashbords.blade.php [hunk 3]
```diff
@@ -0,0 +1,295 @@
+@extends('layouts.app')
```
Comment (kredo-school) [inline resources/views/admin/dashbords.blade.php]: please correct also the spelling of the file name :). It should be `dashboard`

File: resources/views/admin/dashboard.blade.php [hunk 4]
```diff
@@ -0,0 +1,94 @@
+@extends('layouts.app')
+
+@section('content')
+
+@section('styles')
```
Comment (kredo-school) [inline resources/views/admin/dashboard.blade.php]: move section styles and script outsite section content

**Review by kredo-school — CHANGES_REQUESTED** [review]

File: routes/web.php [hunk 5]
```diff
@@ -17,6 +17,9 @@
 Route::get('/', function () {
     return view('index');
 });
+Route::get('admin/dashboard', function () {
```
Comment (kredo-school) [inline routes/web.php]: add `/` before `admin/dashboard`

**Review by kredo-school — CHANGES_REQUESTED** [review]

File: resources/views/admin/dashboards.blade.php (hunk 1 above)
Comment (Tommmy1018) [reply resources/views/admin/dashboards.blade.php]: I did.

File: resources/views/admin/dashboards.blade.php (hunk 2 above)
Comment (Tommmy1018) [reply resources/views/admin/dashboards.blade.php]: I did

File: resources/views/admin/dashbords.blade.php (hunk 3 above)
Comment (Tommmy1018) [reply resources/views/admin/dashbords.blade.php]: I modified it

File: routes/web.php (hunk 5 above)
Comment (Tommmy1018) [reply routes/web.php]: I fixed it

File: app/Http/Controllers/HomeController.php [hunk 6]
```diff
@@ -46,7 +46,12 @@ public function profileReservation(){
         return view('users.profilereservation');
     }
 
+<<<<<<< HEAD
```
Comment (kredo-school) [inline app/Http/Controllers/HomeController.php]: @Tommmy1018 You included in your commit the conflict.

Comment (Tommmy1018) [reply app/Http/Controllers/HomeController.php]: What does that mean? I couldn't find any conflict in the file.

**Review by kredo-school — DISMISSED** [review]

**Review by Ditzler01 — DISMISSED** [review]

**Review by Ditzler01 — APPROVED** [review]

**Pull request merged**